src/core/expr/fexpr.cc

```diff
@@ -489,7 +489,7 @@ DECLARE_METHOD(&PyFExpr::min)
 
 oobj PyFExpr::nth(const XArgs& args) {
   auto nthFn = oobj::import("datatable", "nth");
-  oobj n = args[0].to_oobj();
+  oobj n = args[0].to_oobj() ? args[0].to_oobj() : py::oint(0);
```

Comment:
Also, space between the condition and `?` for ternary operators is not really needed. Just like in the normal sentence you wouldn't put the space between a word and `?`.